File: test/integration_tests/release/data/subscribed_project/.prophecy/1665/pipelines/Delta/code/src/main/scala/io/prophecy/pipelines/delta/graph/Filter_1.scala
package io.prophecy.pipelines.delta.graph

import io.prophecy.libs._
import io.prophecy.pipelines.delta.udfs.PipelineInitCode._
import io.prophecy.pipelines.delta.udfs.UDFs._
import io.prophecy.pipelines.delta.config.Context
import org.apache.spark._
import org.apache.spark.sql._
import org.apache.spark.sql.functions._
import org.apache.spark.sql.types._
import org.apache.spark.sql.expressions._
import java.time._

object Filter_1 {
  def apply(context: Context, in: DataFrame): DataFrame = in.filter(lit(true))
}

File: test/resources/BaseDirectory/pipelines/FirstPipeline/code/firstpipeline/graph/Filter_1.py
from pyspark.sql import *
from pyspark.sql.functions import *
from pyspark.sql.types import *
from prophecy.utils import *
from prophecy.libs import typed_lit
from firstpipeline.config.ConfigStore import *
from firstpipeline.udfs.UDFs import *

def Filter_1(spark: SparkSession, in0: DataFrame) -> DataFrame:
    return in0.filter(lit(True))
